src/Views/Movie/_Form.cshtml

```diff
@@ -0,0 +1,43 @@
+@using Microsoft.AspNetCore.Mvc.Localization
+@using System.Globalization;
+@inject IViewLocalizer Localizer
+@model TrainingNet.Models.Views.MovieViewModel
+
+@using (Html.BeginForm("@ViewData[\"FormAction\"].Value","Movie",FormMethod.Post, new { @class = "form-horizontal" }))
+
+{  
+    @Html.AntiForgeryToken()
+    <br>
+    <h4>@Localizer["Movie"]</h4>
+    <hr/>
+        @Html.ValidationSummary("", new { @class = "text-danger" })
+        <div class = "form-group">
+            @Html.Label("Title",Localizer["Movie"].Value, new { @class = "col-md-2 control-label" })
+            <div class = "col-md-10">
+                @Html.TextBoxFor(m => m.Title, new { @class = "form-control" })
+            </div>
+        </div>
+        <div class = "form-group">
+            @Html.Label("ReleaseDate",Localizer["ReleaseDate"].Value, new { @class = "col-md-2 control-label" })
+            <div class = "col-md-10">
+                @Html.TextBoxFor(m => m.ReleaseDate, new { @class = "form-control" })
```

Comment:
Try using EditorFor instead of TextBoxFor

Reply:
I will take your advice on the following Card